### PR TITLE
Feature/eip1 4398 liquibase scripts to create table for approved postal/proxy application

### PR DESCRIPTION
**Pull request by spalamthodi**
The Liquibase scripts for approved postal/proxy application tables.

https://technologyprogramme.atlassian.net/browse/EIP1-4398
https://technologyprogramme.atlassian.net/browse/EIP1-4289

### CODE REVIEW

File: src/main/resources/db/changelog/create/0004_approved_proxy_application.sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    `registered_address_id`     char(36)     NOT NULL,
+    `reference_number`          varchar(10)  NOT NULL,
+    `ip_address`                varchar(45)  NOT NULL,
+    `ems_elector_id`            varchar(255) NOT NULL,
+    `language`                  char(2),
+    `proxy_first_name`          varchar(35)  NOT NULL,
+    `proxy_middle_names`        varchar(100),
+    `proxy_surname`             varchar(35)  NOT NULL,
+    `proxy_family_relationship` varchar(500),
+    `proxy_address_id`          char(36),
```
Comment (inderjitvirdi) [inline src/main/resources/db/changelog/create/0004_approved_proxy_application.sql]: Is proxy address mandatory and so should this be not null?

File: src/main/resources/db/changelog/create/0004_approved_proxy_application.sql [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    `proxy_reason`              varchar(500),
```
Comment (inderjitvirdi) [inline src/main/resources/db/changelog/create/0004_approved_proxy_application.sql]: proxy reason also mandatory?

File: src/main/resources/db/changelog/create/0001_revinfo.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+-- liquibase formatted sql
+-- changeset Krister.Bone@valtech.com:create-revinfo-table endDelimiter:;
+
+CREATE TABLE `revinfo`
```
[the remark this answers is not in the document]
Comment (spalamthodi) [reply src/main/resources/db/changelog/create/0001_revinfo.sql]: Already removed

File: src/main/resources/db/changelog/create/0004_approved_proxy_application.sql (hunk 1 above)
Comment (spalamthodi) [reply src/main/resources/db/changelog/create/0004_approved_proxy_application.sql]: Done

File: src/main/resources/db/changelog/create/0004_approved_proxy_application.sql (hunk 2 above)
Comment (spalamthodi) [reply src/main/resources/db/changelog/create/0004_approved_proxy_application.sql]: Done

**Review by inderjitvirdi — APPROVED** [review]

**Pull request merged**